src/backend/lib/queue.js

```diff
@@ -31,7 +31,15 @@ function createQueue(name) {
   })
     .on('error', err => {
       // An error occurred
-      logger.error({ err }, `Queue ${name} error`);
+      if (err.code === 'ECONNREFUSED') {
+        logger.info(
```

Comment:
Probably this should still remain an error: `logger.error` vs. `logger.info`

Reply:
Sure, change made: https://github.com/Seneca-CDOT/telescope/pull/553/commits/ffcebe5070a824d3942778229e4dc60d2f2b55e6